.github/workflows/e2e-test-daily.yml

```diff
@@ -84,6 +84,9 @@ jobs:
           azureIAMCreateCredentials: ${{ secrets.AZURE_E2E_IAM_CREDENTIALS }}
           registry: ghcr.io
           githubToken: ${{ secrets.GITHUB_TOKEN }}
+          awsOpenSearchDomain: ${{ secrets.AWS_OPENSEARCH_DOMAIN }}
+          awsOpenSearchUsers: ${{ secrets.AWS_OPENSEARCH_USER }}
+          awsOpenSearchPwd: ${{ secrets.AWS_OPENSEARCH_PWD }}
```

Comment:
Don't we need this for the other e2e workflows as well?

Reply:
All other workflows should already have it

Reply:
update: added it to release e2e test where it was missing